### PR TITLE
Lecture1 exercises

### PR DESCRIPTION
**Pull request by Mugenor**
### Solutions for Lecture 1

cc @chshersh

### CODE REVIEW

File: src/Lecture1.hs [hunk 1]
```diff
@@ -54,7 +56,9 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Num a => a -> a -> a
```
Comment (chshersh) [inline src/Lecture1.hs:59]: `Num` is explained in the following lectures, so you are one step ahead 😸
In here `Int -> Int -> Int` or `Integer -> Integer -> Integer` would also work, as a more specific type 🙂

**Review by chshersh — APPROVED** [review]
Congrats! The first homework is complete 👏🏼 
Hope you enjoyed the course so far. Keep up the great work, and feel free to ask any questions if you have 👌🏼

Your solutions to this lecture already look great, and I don't have anything to add 🙂 

<hr>

It would mean a lot to me if you [star my course on GitHub](https://github.com/haskell-beginners-2022/course-plan/stargazers) ⭐ (if you already haven't done so)! 🤗

File: src/Lecture1.hs (hunk 1 above)
Comment (Mugenor) [reply src/Lecture1.hs:59]: At this step I didn't understand completely what I'm doing until I watched the following lectures :)

**Comment by Mugenor** [comment]
Thank you for your effort!

**Pull request merged**